Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.14.1-alpine
+FROM nginx:1.15.12-alpine
```

Comment:
Out of curiosity: How does this `nginx` container differ from having a "standard" alpine container with nginx installation step?

Reply:
No idea. We use `nginx` for convenience reasons. Feel free to inspect the image's Dockerfile for details. https://github.com/nginxinc/docker-nginx/blob/e5123eea0d29c8d13df17d782f15679458ff899e/mainline/alpine/Dockerfile